src/components/Badge/Badge.scss

```diff
@@ -71,6 +71,17 @@ $pip-spacing: ($height - $pip-size) / 2;
   }
 }
 
+.statusCritical {
+  @include pip-color(var(--p-icon-warning, color('red', 'dark')));
```

Comment:
This uses the same color as "warning" in the new design system - you'll need to update that

Reply:
```suggestion
  @include pip-color(var(--p-icon-critical, color('red', 'dark')));
```